admin/templates/configuration/etc/init.d/xrootd

```diff
@@ -74,12 +74,11 @@ start_service() {
     pidfile=$(pid_file_name $prog)
     lockfile=${QSERV_RUN_DIR}/var/lock/subsys/${prog}
     BINARY=${XROOTD_DIR}/bin/${prog}
-#   logfile=${LOG_DIR}/${prog}.log
```

Comment:
It was commented, so no more used.
Its current value is: 
logfile=@libXrdSsiLog.so